core/runtime.rs

```diff
@@ -361,7 +361,7 @@ impl JsRuntime {
       .into_iter()
       .enumerate()
       .map(|(id, decl)| {
-        OpCtx::new(id, 0, Rc::new(decl), op_state.clone(), weak.clone())
+        OpCtx::new(id as u16, 0, Rc::new(decl), op_state.clone(), weak.clone())
```

Comment:
Why not make `OpId` u16 in `OpCtx`?

Reply:
It is, when you enumerate you get `usize` for the index